crates/kubelet/src/status.rs

```diff
@@ -0,0 +1,122 @@
+//! Container statuses
+use chrono::{DateTime, Utc};
+use k8s_openapi::api::core::v1::ContainerStatus as KubeContainerStatus;
+use k8s_openapi::api::core::v1::{
+    ContainerState, ContainerStateRunning, ContainerStateTerminated, ContainerStateWaiting,
+};
+use k8s_openapi::apimachinery::pkg::apis::meta::v1::Time;
+
+use std::collections::HashMap;
+
+/// Describe the status of a workload.
+///
+/// Phase captures the lifecycle aspect of the workload, while
```

Comment:
This comment is no longer valid as we don't capture phase directly. It should be updated to reflect that container statuses is part of it